throw
0.6〜
-----------------------------------------------------------
[ 機能 ]
例外処理 catch を実行します。

[ 書式 ]
throw <値>

[ 例 ]
throw 2;

[ 解説 ]
try 内で記載します。

